models/SlackAddForm.php

```diff
@@ -91,7 +91,7 @@ public function validateWebhookUrl($attr, $params)
 
             // Discord
             sprintf('/^%s/ui', implode('', [
-                $quote('https://discordapp.com/api/webhooks/'),
+                $quote('https://discord.com/api/webhooks/'),
```

Comment:
Thank you for submitting the pull request.
Please make some changes to the code to make it better and more consistent with the policies of this repository.

- Please change to also accept older URL.
  maybe
  ```php
    sprintf('/^%s/ui', implode('', [
        $quote('https://'),
        vsprintf('(?:%s|%s)', [
            $quote('discord.com'),
            $quote('discordapp.com'),
        ]),
        $quote('/'),
        '\d+', // snowflake
        $quote('/'),
        '[0-9A-Za-z_-]+',
        $quote('/slack'),
    ])),
  ```
  is much better.
- Please add your name and email address to file header. (Please refer [the CI result](https://github.com/fetus-hina/stat.ink/runs/2161735396?check_suite_focus=true))
  If you would like to put a different name or email address, you will also need to change the following file: [`/.check-author.yml`](https://github.com/fetus-hina/stat.ink/blob/ca0fbd56175461d7f3fd76f8dfe00a5d6022b541/.check-author.yml#L1-L7)